src/cli.rs

```diff
@@ -88,6 +90,9 @@ pub enum Command {
     // ---------------- Libp2p Tools ----------------------------//
     P2pPing(PingArgs),
     P2pIdentify(IdentifyArgs),
+
+    // ---------------- CAS Tools ----------------------------//
+    AnchorRequest(AnchorRequestArgs),
```

Comment:
What does this AnchorRequest do? Can you give it its own comment in addition to the separator?